src/index.js

```diff
@@ -1,32 +1,75 @@
 /* eslint-disable no-unused-vars */
+
 import _ from 'lodash';
 import './style.css';
+import bin from './bin.svg';
+
+import { addNewComponent, removeComponents } from './modules/functions.js';
 
-const components = [
-  {
-    description: 'complete to Do list project',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'wash the dishes',
-    completed: false,
-    index: 0,
-  },
-];
-
-const showComponents = () => {
+const showComponents = (components) => {
   const container = document.querySelector('.to-do-list');
+  container.innerHTML = ''; // Clears the container
 
   // Sort components array based on index property
   components.sort((a, b) => a.index - b.index);
 
   components.forEach((component) => {
     const items = document.createElement('li');
     items.className = 'list-item';
-    items.innerHTML = `<input class="check-box" type="checkbox">${component.description}`;
+    items.dataset.index = component.index;
+    items.innerHTML = `
+    <div class="listboxflex">
+        <input class="check-box" type="checkbox">
+        <input class="description" value="${component.description}" readonly>
+        <a class="delete-icon"><img class="delete-img" src="${bin}" alt="bin"></a>
```

Comment:
- [ ] As mentioned by the previous reviewer this image is broken, please kindly fix that 🙏🏾 

<img width="857" alt="Screenshot 2023-07-15 at 00 47 15" src="https://github.com/Bupilipili/To-do-list/assets/74501523/4facc9f3-fab2-4c39-8c09-ccb3172c115e">

- [ ] Also the index of elements in the array should start with 1 and not 0, (i.e. if a 4th task is added to the list, the index of that task should equal to 4 and not 3). Is part of the project requirements the first reviewer forgot to mention that, please kindly fix that 🙏🏾 

<img width="1023" alt="Screenshot 2023-07-15 at 01 09 04" src="https://github.com/Bupilipili/To-do-list/assets/74501523/87c92e51-4d38-4bba-8305-884efd8ab68e">

<img width="371" alt="Screenshot 2023-07-15 at 00 45 30" src="https://github.com/Bupilipili/To-do-list/assets/74501523/27efba81-08d3-4774-8075-95ea5923e7e4">